script/Ciel/Day 13/13_01_02_CIEL13_6.txt

```diff
@@ -296,77 +297,78 @@ On the other hand, it seems that the enemy is in a similar position.
 -- Page 128, Offset 35618.
 -- 肘に弾き飛ばされただけで右肩の骨が粉砕した。
-The bones on my right shoulder are smashed from just being blown away by her elbow.
+Even though she only hit me with her elbow, the bones on my right shoulder have been pulverized.
 }
 [sha:60016993b7824479e25f41e68c62812b11d2f5f9]{
 -- Page 129, Offset 35619.
 -- 焼くような痛みは傷の痛みではなく、傷を治す痛みだ。
-This scorching pain is not coming from the wound itself, but from the pain of healing it.
+This scorching pain isn't coming from the wound itself, but rather because my body is trying to heal it.
 }
 [sha:4b55aafb4150b19ba462d1176ea8d5c3df7e46c3]{
 -- Page 129, Offset 35620.
 -- 見なくても分かる。いま肩の内側では、砕け散った骨をヒルのような筋肉が集め、包み、強引に“元に戻して”いる。
-I can tell without even looking. Right now, the muscles are gathering around the broken bones like a hill, wrapping around it and forcing it to 'return back to normal'.
+I can tell without even looking. Within my shoulder, the muscles are congealing around the broken bones like a swarm of leeches, wrapping around them and forcefully restoring them back to their original state.
 }
 [sha:32c5d726da3d21413fcc23464385a5f3953960d9]{
 -- Page 130, Offset 35621.
 -- 「―――なあ。さっきから、気になってたんだけど」
-"―――Hey. I've been curious about something for a while."
+"―――Hey. There's something I've been curious about for a while."
 }
 [sha:9dd92c8a25c02385949641037a508f8d7d1452db]{
 -- Page 130, Offset 35622.
 -- 数秒でいい。せめて肩が繋がるまで時間を稼ぎたい。
-I just need a few seconds. I need to buy enough time to connect my shoulders at the very least.
+A few seconds will do. I need to buy enough time for my shoulder to reconnect at the very least.
 }
 [sha:082507380dfadaec8531e3a0c33da8e073028407]{
 -- Page 131, Offset 35623.
 -- 「なに？　わたしの魅力にまいっちゃった？　年下の先生って興奮しちゃう？」
-"What is it? Have you fallen for my charms? Are you excited from seeing your teacher being younger than you?"
+"What is it? Have you been captivated by my charms? Are you getting excited from looking at a teacher who's younger than you?"
 }
 [sha:b8ac4ef4f3703b03935f67549a39ecc4b897cf0a]{
 -- Page 131, Offset 35624.
 -- 「……そりゃあ、別の意味でまいってるけど」
-"...Well, you could say I've fallen for something in a different sense."
+"...Well, I guess I am captivated, in a completely different sense."
 }
 [sha:115d3e909de66fcd1d894e4efe3614fa34aabca7]{
 -- Page 132, Offset 35625.
 -- 死徒とは“生命のバージョンアップ”だ。
-One could argue that a Dead Apostle is an 'upgraded version of a life'.
+One could argue that a Dead Apostle is an 'upgraded lifeform'.
 }
 [sha:4146e19067238612f024c6db8a7391721bfaa114]{
 -- Page 132, Offset 35626.
 -- 吸血鬼になる事でその身体の最適解ともいえる美貌を得る事もあるし、人間以上の快楽を与える名器にもなり得る。
-By turning into a vampire, you can achieve the highest level of beauty for that body, and you could also become a masterpiece that offers more pleasure than any other human being.
+// nasu really had to write about hitting the vampussy didnt he
+Upon becoming a vampire, one's body morphs into the most beautiful form it can possibly take, creating a masterpiece that is more pleasurable than any human being.
```

Comment:
this is one of those lines i can't wait to see posted everywhere and scrutinized. gonna be fun!